[2025-06-09 18:44:44,969](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.168', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021F7C6C75E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-06-09 18:50:28,060](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.168', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000016E112385E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-06-09 18:55:42,970](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.168', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000021807C585E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-06-09 19:02:32,648](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.168', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000019DDEB885E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-06-09 19:05:33,633](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.168', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AFDA2395E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
[2025-06-09 19:25:07,451](__post_init__) Проблема с подключением к серверу очереди: HTTPConnectionPool(host='10.0.5.168', port=32107): Max retries exceeded with url: /get_wplace?w=8 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E9151885E0>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))
